SPECS/mariner-rpm-macros/generate-package-note.py

```diff
@@ -49,20 +49,23 @@
         --version "1.2.3.4" --moduleVersion "1.2.3.4-beta" \\
         --os "mariner" --osVersion "1.0" \\
         --maintainer "DEADBEEF-BBC3-4FED-9192-110C11DED04D" \\
-        --copyright "Microsoft" --repo "CBL-Mariner" \\
+        --copyright "Microsoft" \\
+        --outdir "/source/Mariner/pkgname/build/" \\
```

Comment:
why isn't this "/usr/src/mariner/BUILD/" ?

Reply:
Changed it to a generic example